tests/components/deconz/test_light.py

```diff
@@ -59,6 +59,16 @@
         "state": {"reachable": True},
         "uniqueid": "00:00:00:00:00:00:00:02-00",
     },
+    "4": {
+        "hascolor": False,
+        "manufacturername": "MLI",
+        "modelid": "tint Smart Switch",
```

Comment:
Shouldn't this switch be … a switch ?

Reply:
I will ask the issuer. According to his issue its a light but I'll verify

Reply:
I will change it to a switch. Thanks 👍🏻